pkg/services/user/userimpl/user.go

```diff
@@ -235,6 +235,16 @@ func (s *Service) GetByUID(ctx context.Context, query *user.GetUserByUIDQuery) (
 	return s.store.GetByUID(ctx, query.UID)
 }
 
+func (s *Service) ListByIdOrUID(ctx context.Context, uids []string, ids []int64) ([]*user.User, error) {
+	ctx, span := s.tracer.Start(ctx, "user.ListByIdOrUID", trace.WithAttributes(
```

Comment:
```suggestion
	if len(uids) == 0 && len(ids) == 0 {
		return []*user.User{}, nil
	}
	ctx, span := s.tracer.Start(ctx, "user.ListByIdOrUID", trace.WithAttributes(
```
In case none are set - would be good to avoid the db call